lib/project_types/extension/commands/serve.rb

```diff
@@ -3,14 +3,20 @@
 module Extension
   module Commands
     class Serve < ExtensionCommand
+      options do |parser, flags|
+        parser.on("--tunnel=TUNNEL") { |tunnel| flags[:tunnel] = tunnel }
```

Comment:
Please use

```ruby
parser.on("-t", "--[no-]tunnel", "Establish ngrok tunnel") { ... }
```

This creates a very idiomatic command line interface that allows you to run the command as follows:

```shell
$ shopify server -t # => `flags[:tunnel]` will be `true`
$ shopify serve --tunnel # => `flags[:tunnel]` will be `true`
$ shopify serve --no-tunnel # => `flags[:tunnel]` will be `false`
$ shopify serve # => `flags[:tunnel]` will be `nil`
```

Ideally, we substitute `nil` with `true` but respect of course when someone runs the command with `--no-tunnel`.